packages/application-shell/package.json

```diff
@@ -94,6 +87,22 @@
     "uuid": "3.3.2"
   },
   "devDependencies": {
+    "@commercetools-frontend/ui-kit": "^2.0.0-rc.0",
+    "react": "16.5.2",
+    "react-apollo": "2.2.4",
+    "react-dom": "16.5.2",
+    "react-intl": "2.7.1",
+    "react-router": "4.3.1",
+    "react-router-dom": "4.3.1",
     "wait-for-observables": "1.0.3"
+  },
+  "peerDependencies": {
+    "@commercetools-frontend/ui-kit": ">=2 <3 || >=2.0.0-rc.1 <3",
```

Comment:
My brain.

Reply:
😅 I added a screenshot with some `semver` tests in case you want to help your brain 😉